tests/vlan/test_autostate_disabled.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import pytest
+
+from tests.common.utilities import wait_until
+
+
+pytestmark = [
+    pytest.mark.topology("t0")
```

Comment:
@Blueve Do you think t0 is a correct topology mark for this test case?

Reply:
Yes

Reply:
OK, thanks!